documentation/adr/0001-recording-architecture-decisions.md

```diff
@@ -0,0 +1,79 @@
+# Recording Architecture Decisions
+
+- **Status:** Proposed
+- **Last Modified:** 2023-06-07
+- **Related Issue:** TODO
+- **Deciders:** TODO
+- **Tags:** ADRs
+
+## Context and Problem Statement
+
+Important decisions about the structure of a codebase, the tools or platforms adopted, and the scope or vision of a product are made all throughout a project's lifecycle. Often these decisions are made and changed blindly with little record of why that decision was made, making it difficult for other collaborators and even the original decision makers to understand the justification behind that decision in the future.
+
+_What is the best way to document key architectural decisions made within the project so that future contributors can understand the justification for those decisions?_
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- **Explicit:** Important decisions about the project architecture should be clear and unambiguous
+- **Documented:** There should be a written record describing which decisions are made
+- **Accessible:** Decision records should be centrally located, searchable, and accessible to non-technical audiences
+- **Portable:** The decision records should be able to travel with the code when it's forked, cloned, etc.
+- **Maintainable:** The system we adopt should be easy to maintain or amend as needed.
+
+## Options Considered
+
+- ADRs in the repo `docs/` folder
+- Comments and docstrings in the code itself
+- Articles in the project Wiki tab
+
+## Decision Outcome <!-- REQUIRED -->
+
+Decisions will be documented using Architecture Decision Records (ADRs) as described by [Joel Parker Henderson](joel) and the [ADR GitHub organization](https://adr.github.io/). The template for this project's ADRs will be adapted from the [MADR template](adrs).
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- ADRs travel with the repo when it's cloned, forked, etc.
+- ADRs can be incorporated in the Issue and PR workflow
+- Changes to ADRs will be listed in the project's commit history
+- ADRs written in Markdown are accessible to non-technical audiences
```

Comment:
Yeah this is a fair point. I was considering it accessible from reading the resulting ADRs (especially compared to inline code comments or docstrings). But in order to make it more accessible to write ADRs we'd probably need some kind of GUI wrapper, which again relates to your suggestion below:

> I might be tempted to vote for an alternative we haven't considered: putting the ADRs in a separate wiki platform (whatever we end up using for our wiki) rather than in the repo, but I could be persuaded either way. Let's discuss verbally sometime.

Reply:
I wound up changing this from a pro to a con for the current decision. I'm hoping there's a way we can have our cake and eat it to with the wiki platform choice, but for now having them be stored and edited in GitHub is probably more of a con from readability. 

This change was made in [this commit](https://github.com/HHS/grants-api/pull/15/commits/0e4207784335a5e4e8a6da53ce7fa979b9ea075e)